PairedSingleSampleWF.nf

```diff
@@ -663,10 +663,8 @@ process multiqc {
 
     script:
     prefix = fastqc[0].toString() - '_fastqc.html' - 'fastqc/'
-    rtitle = custom_runName ? "--title \"$custom_runName\"" : ''
-    rfilename = custom_runName ? "--filename " + custom_runName.replaceAll('\\W','_').replaceAll('_+','_') + "_multiqc_report" : ''
     """
-    multiqc -f $rtitle $rfilename --config $multiqc_config . 2>&1
+    multiqc -f . 2>&1
```

Comment:
Aww, shame! Is it not just as easy to copy in this stuff from another pipeline? I quite like the custom report title and filename, plus the config is useful for stuff like putting the software version numbers at the bottom of the report.

Phil

Reply:
Ah, #19 I guess.. Sorry, reading emails in chronological order..

Reply:
Hi Phil! 
I wanted to keep it but had quite bad issues because the `multiqc_config.yaml` was not copied in (as for example in the NGI-RNAseq or CAW pipelines). I couldn't really find a solution to that.

Reply:
Can find it here, but when I call the singularity image (for multiqc, which normally produces a working report without the -c/--config parameter), it results in an error:

```
ls -la
total 1166
drwxr-xr-x 8 iiipe01 ii      17 Jan 15 14:23 .
drwxr-xr-x 4 iiipe01 ii       2 Jan 15 14:23 ..
-rw-r--r-- 1 iiipe01 ii       0 Jan 15 14:23 .command.begin
-rw-r--r-- 1 iiipe01 ii     322 Jan 15 14:23 .command.err
-rw------- 1 iiipe01 ii     896 Jan 15 14:23 .command.log
-rw-r--r-- 1 iiipe01 ii     574 Jan 15 14:23 .command.out
-rw-r--r-- 1 iiipe01 ii    5079 Jan 15 14:23 .command.run
-rw-r--r-- 1 iiipe01 ii      44 Jan 15 14:23 .command.sh
-rw-r--r-- 1 iiipe01 ii    2666 Jan 15 14:23 .command.stub
-rw-r--r-- 1 iiipe01 ii       2 Jan 15 14:23 .command.trace
-rw-r--r-- 1 iiipe01 ii       1 Jan 15 14:23 .exitcode
drwxr-xr-x 2 iiipe01 ii       4 Jan 15 14:23 fastqc
drwxr-xr-x 2 iiipe01 ii       1 Jan 15 14:23 gatk_base_recalibration
lrwxrwxrwx 1 iiipe01 ii      78 Jan 15 14:23 multiqc_config.yaml -> /home/tu/iiipe01/.nextflow/assets/apeltzer/NGI-ExoSeq/conf/multiqc_config.yaml
drwxr-xr-x 2 iiipe01 ii       6 Jan 15 14:23 multiqc_data
-rw-r--r-- 1 iiipe01 ii 1176125 Jan 15 14:23 multiqc_report.html
drwxr-xr-x 2 iiipe01 ii       1 Jan 15 14:23 qualimap
drwxr-xr-x 2 iiipe01 ii       1 Jan 15 14:23 software_versions
drwxr-xr-x 2 iiipe01 ii       2 Jan 15 14:23 trimgalore
```

Reply:
I'm creating the `multiqc_config.yaml` on the go when needed, I found that it was a good solution

Reply:
Running something like  `singularity exec multiqc.img multiqc --config multiqc_config.yaml` tells me that there is no such file (while there is...) .

Reply:
I'll try to do that now too

Reply:
Strange - I can't see any reason why it should work in one pipeline and not the other...?

Reply:
Hmm, I have this here:

```
multiqc_config.yaml -> /home/tu/iiipe01/.nextflow/assets/apeltzer/NGI-ExoSeq/conf/multiqc_config.yaml
```
Which points to a path not accessible from inside the container, since the homes are here:
```
/home-link/iiipe01/.... 
``` 

Maybe something misconfigured with singularity?

Reply:
Locally it works. Maybe I could follow the approach from @MaxUlysse, creating the file on demand and thus making this path issue not happening again. Should be safer I suppose.

Reply:
Okay, I followed @MaxUlysse s approach and we have a pretty report now again :-) 
[MQC(1).zip](https://github.com/SciLifeLab/NGI-ExoSeq/files/1633057/MQC.1.zip)

Reply:
Nice work! Personally I think that chunk at the top would look better as a report section, in the same style as the software versions. I'm not sure that it deserves such a high priority :) Could wrap the command in `<pre>command</pre>` to make it look nicer too.